spec/concurrent/set_spec.rb

```diff
@@ -55,6 +55,33 @@ module Concurrent
         end.map(&:join)
         expect(set).to be_empty
       end
+
+      it 'force context switch' do
```

Comment:
writing a test for this is actually not easy lol.

What this ended up testing was that during a method of set it grabs a monitor and locks out any other changes.
```ruby
expect(set.add?(1)).to eq set
```
the above line _should_ occasionally return **nil** if the methods were not synchronized.